smithy-model/src/test/resources/software/amazon/smithy/model/errorfiles/validators/event-stream-validator.json

```diff
@@ -43,35 +39,31 @@
 
       "ValidMultiEventInputOperation": {
         "type": "operation",
-        "input": "ValidMultiEventInput",
-        "inputEventStream": "events"
+        "input": "ValidMultiEventInput"
       },
 
       "ValidMultiEventOutputOperation": {
         "type": "operation",
-        "output": "ValidMultiEventOutput",
-        "outputEventStream": "events"
+        "output": "ValidMultiEventOutput"
       },
 
       "ValidMultiEventBidirectionalOperation": {
         "type": "operation",
         "input": "ValidMultiEventInput",
-        "inputEventStream": "events",
-        "output": "ValidMultiEventOutput",
-        "outputEventStream": "events"
+        "output": "ValidMultiEventOutput"
       },
 
       "ValidMultiEventInput": {
```

Comment:
The structures for `ValidMultiEvent[Input|Output]` should target the `ValidUnion` type, not `Event`